usr/share/device-quirks/scripts/asus/ally/ally.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+if [ $(whoami) != 'root' ]; then
+  echo "You must be root to run this script."
+  exit 1
+fi
+
+if [ -d /tmp/frzr_root ]; then
+  source ${MOUNT_PATH}/etc/device-quirks/device-quirks.conf
+else
+  source /etc/device-quirks/device-quirks.conf
+fi
```

Comment:
7-11 are not necessary as the applies quirk does not invoke a firmware override

Reply:
Thank you for your advice!
Delete the appropriate line.

Reply:
Removed firmware override processing.